src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { Box, Checkbox, Flex, Sans, Toggle } from "@artsy/palette"
+import { useArtworkFilterContext } from "../ArtworkFilterContext"
+
+const sizeMap = [
+  { displayName: "Small (under 40cm)", name: "SMALL" },
+  { displayName: "Medium (40 – 100cm)", name: "MEDIUM" },
+  { displayName: "Large (over 100cm)", name: "LARGE" },
+]
+
+/**
+ * Note: This is a clone of the implementation at
+ * src/v2/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/SizeFilter.tsx
+ * adapted to work with the common ArtworkFilter.
+ */
```

Comment:
This ended up being cloned because the auction results version of the SizeFilter is couple with `AuctionResultsFilterContext` which has a different shape/purpose/query than these shared artwork filters. 

Trying to merge them was too big a lift, though @pvinis is interested in attempting that.

Reply:
It _seems_ the only difference is `.filters` vs `.getCurrentFilters`, no?

Reply:
That's what I thought at first glance too! 

Pavlos and I paired on this briefly, and it turned into a cascade of things that were present in `ArtworkFilterContext` but not in `AuctionResultsFilterContext` — all the `stagedFilters` and associated actions and reducers that let you actually get a meaningful result from `. getCurrentFilters()`

Reply:
Ah I see, np then 👍